.github/workflows/scripts/distroless.pub

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
@loosebazooka 

Hope we can move this to SLSA :)

Reply:
That's for signing images directly with cosign. There is some space to say "slsa attestation are enough, you don't need to double sign images"

Reply:
yeah, if distroless did releases on GHA we could have them use the generic container workflow and then we could check the SLSA attestation.